Parsedown.php

```diff
@@ -553,7 +553,7 @@ protected function blockHeader($Line)
         }
 
         $text = trim($text, ' ');
-        $link = strtolower(str_replace(' ','-',$text));
+	$link = preg_replace('/[^\p{L}\p{N}\p{M}-]+/u', '', mb_strtolower(mb_ereg_replace(' ','-',$text)));
```

Comment:
Do we absolutely need the `mb_ereg_replace()` here? It's not a regular expression, so we should be fine using `str_replace(' ', '-', $text)`. GFM seems to convert consecutive spaces to consecutive dashes as well. 

I'm not a decision maker, but as someone who spends a lot of time processing text and with this library, this looks great to me otherwise. Thank you!

Reply:
I think so, as if you consider a four byte character, one of those could match a space.  Thus it would invalidate that character or change it to something it's not meant to be.  I never deal with them but if we really want to make it universal, I would suggest we do.